slider.css

```diff
@@ -0,0 +1,57 @@
+.slider {
+  display: block;
+}
+
+.slider-container {
+  display: inline-block;
+  position: relative;
+}
+
+.slider-list {
+  padding: 0;
+  margin: 0;
+}
+
+.slider-item {
+  display: none;
+}
+
+.slider-item-active {
+  display: block;
+}
+
+.navigation-buttons {
+  display: none;
+}
+
+.navigation-buttons .btn {
+  position: absolute;
+  top: 50%;
+  transform: translate(0, -50%);
+}
+
+.navigation-buttons .btn.btn-prev {
+  left: 0;
+}
+
+.navigation-buttons .btn.btn-next {
+  right: 0;
+}
+
+.showNavigationButtons {
```

Comment:
another code style. Other classes are named through the dashes "-". This class is camelCase. Please use one code style across the project